test/nlp/test_default.py

```diff
@@ -91,9 +91,13 @@ def test_points_to_evaluate():
     automl_settings = get_automl_settings(estimator_name="transformer_ms")
 
     automl_settings["estimator_list"] = ["transformer_ms"]
-    automl_settings["starting_points"] = "data"
-
+    automl_settings["starting_points"] = "data:test/nlp/default/"
     del automl_settings["fit_kwargs_by_estimator"]["transformer_ms"]["model_path"]
```

Comment:
Why did we have "model_path" in `fit_kwargs_by_estimator` in the beginning? Shall we remove it in `get_automl_settings()`?

Reply:
get automl settings is for providing a low cost config so it includes overwriting the model to Electra small

Reply:
I said `fit_kwargs_by_estimator`. Why does it contain "model_path"?

Reply:
We must overwrite model_path to electra small. Do you mean to overwrite "model_path", why do we do it via fit_kwargs_by_estimator instead of other options (such as fit_kwargs)?